site/docs/components/slider.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Slider
+navTitle: Slider (not built)
+summaryParagraph: A slider lets people imprecisely select a value from a subjective range by moving a handle horizontally.
+tags: ["Range", "Range input", "Range slider"]
+needToKnow:
+- Avoid sliders when choosing a precise value is important.
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### Examples
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D47508%253A35978&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+Sliders are helpful for choosing subjective values.
+
+- Always show the current, selected value.
+- Provide a visible label describing what the slider changes, such as "Brightness".
+- Show the minimum value on the left and maximum value on the right, where these are reversed in Right-To-Left (RTL) languages.
+- We only use horizontal sliders. We're yet to need a vertical slider.
+- We only use sliders with no fill so far.
+- We only use sliders with a single handle. We're unlikely to need a slider with more than one handle.
+
+### Style
+
+- There's rarely a need to apply labels to every tick mark. Consider only showing a label for min, max, and middle ticks, such as 0%, 100%, and 50%.
```

Comment:
It looks like we use ticks on the new 1-1 in Performance -- I just did a quick look. I wonder if that is more feedback for that slider? I think Performance needs to up its standards to align with this doc.

![Screen Shot 2020-08-05 at 4 39 50 pm](https://user-images.githubusercontent.com/4946487/89380124-59a0b200-d73a-11ea-8761-7a29d88ede57.png)

Reply:
Ticks is fine, tick *labels* is the key detail here. Updated the docs to try make it clearer.